moja-kuhinjica-frontend/src/pages/index.tsx

```diff
@@ -76,6 +73,30 @@ const Home = (): JSX.Element => {
         }
     }, [windowWidth])
 
+    const sortMenusByDate = (menus: IMenu[]): IMenu[] => {
+        return menus.sort((a: IMenu, b: IMenu) => {
+            const dateA = new Date(a.date)
```

Comment:
better naming dateA and dateB is hard to read and understand code

Reply:
change done